### PR TITLE
DOC: Added perspective README

### PR DESCRIPTION
**Pull request by yarikoptic3**

### CODE REVIEW

**Comment by jeremymanning** [comment]
pull request denied: typos in readme file

**Comment by jeremymanning** [comment]
jk :)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Summer school at Dartmouth 2017
+
+Glorious people got together in the hills of New Hampshire to study
+and share their knowledge
```
Comment (yarikoptic3) [inline README.md:4]: what hills?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Introduction
+
+contribute here
```
Comment (yarikoptic3) [inline README.md:8]: yes, needs to be done

**Review by yarikoptic3 — COMMENTED** [review]
more goodness

**Pull request merged**